uisdk/src/main/res/layout/uisdk_view_quotes_item_prebook.xml

```diff
@@ -0,0 +1,177 @@
+                        android:textStyle="normal"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:gravity="bottom|end"
+                        android:maxLines="1"
+                        android:textColor="@color/kh_uisdk_text_primary"
+                        android:textSize="@dimen/kh_uisdk_text_size_small"
+                        tools:text="Fixed" />
+                </LinearLayout>
+
+            </LinearLayout>
+
+
+        </LinearLayout>
+
+        <View
+            android:layout_width="match_parent"
+            android:layout_height="0dp"
+            android:background="@color/bt_border_color" />
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:background="@drawable/kh_uisdk_quote_list_item_review_background"
+            android:orientation="horizontal"
+            android:paddingStart="@dimen/kh_uisdk_spacing_semi_small"
+            android:paddingTop="@dimen/kh_uisdk_spacing_small_intermediate"
+            android:paddingBottom="@dimen/kh_uisdk_spacing_small_intermediate"
+            android:paddingEnd="@dimen/kh_uisdk_spacing_semi_small">
+
+            <ImageView
+                android:id="@+id/logoImageSmall"
+                android:layout_width="@dimen/kh_uisdk_spacing_small"
+                android:layout_height="@dimen/kh_uisdk_spacing_small"
+                android:layout_gravity="center"
+                android:scaleType="centerInside" />
+
+            <TextView
+                android:id="@+id/quoteNameText"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="@dimen/kh_uisdk_spacing_xxsmall"
+                android:ellipsize="end"
+                android:maxLines="1"
+                android:textColor="@color/kh_uisdk_text_primary"
+                android:textSize="@dimen/kh_uisdk_text_size_small"
+                tools:text="Quote Name" />
+
+            <LinearLayout
+                android:id="@+id/quoteFleetRatingLayout"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:gravity="end"
+                android:orientation="horizontal">
+
+                <ImageView
+                    android:layout_width="wrap_content"
+                    android:layout_height="match_parent"
+                    android:src="@drawable/uisdk_ic_star" />
+
+                <TextView
+                    android:id="@+id/quoteFleetRating"
+                    android:layout_width="wrap_content"
+                    android:layout_height="match_parent"
+                    android:textColor="@color/kh_uisdk_warning"
+                    android:textSize="@dimen/kh_uisdk_text_size_small"
+                    tools:text="Fleet Rating" />
```

Comment:
This hardcoded text is ok for now given that ratings are always hidden. We just need to make sure that we don't set it to visible until the feature is ready